.github/workflows/e2e.yml

```diff
@@ -0,0 +1,204 @@
+name: SalesforceCommerceCloud/pwa-kit/e2e
+on:
```

Comment:
Will there every be a case that we'll want to trigger the e2e tests manually? Might be worth considering adding `workflow_dispatch` back

Reply:
We'll add it back if at all cron feels unacceptably flaky.